Frends.ServiceBus.Read/Frends.ServiceBus.Read.Test/UnitTests.cs

```diff
@@ -0,0 +1,718 @@
+using Frends.ServiceBus.Read.Definitions;
+using Microsoft.Azure.ServiceBus;
+using Microsoft.Azure.ServiceBus.Core;
+using Microsoft.Azure.ServiceBus.InteropExtensions;
+using Microsoft.Azure.ServiceBus.Management;
+using System.Net.Mime;
+using System.Text;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Frends.ServiceBus.Read.Test
+{
+    [TestClass]
+    public class UnitTests
```

Comment:
This is pretty large file. You could split test on different files relating on what they are testing.

Reply:
It's sure is large because of the sending operations but if you collapse this sending part (region Send test message) you see that it's not that large anymore.  :)